code/game/objects/items/weapons/clown_items.dm

```diff
@@ -121,12 +121,13 @@
 
 /obj/item/weapon/bikehorn/attack_self(mob/user)
 	if(!spam_flag)
-		spam_flag = 1
+		spam_flag = TRUE

```

Comment:
Unstated change in PR have re and just below with a addtimer. Probably fine though.